doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -195,19 +285,548 @@ Mail chapel-developers@ for review and merge
 
 * For the day or two after the commit has gone in, check the
   chapel-test-results-regressions_ mailing list to ensure that there are no new
-  failures (or that they are due to a commit other than yours).  Use the
-  chapel-developers_ mailing list if you are unsure (a member of the core
-  Chapel team will be tasked with diagnosing any testing failures on any given
-  night, but it's nice when developers notice the issue first themselves to
-  save wasted effort).
+  failures caused by your commit.  Use the chapel-developers_ mailing list if
+  you are unsure (a member of the core Chapel team will be tasked with
+  diagnosing any testing failures on any given night, but it's nice when
+  developers notice the issue first themselves to save wasted effort).
+
+.. _HOWTO/Git/GitHub details:
+
+HOWTO/Git/GitHub details
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _initial git setup:
+
+Initial Git Setup
++++++++++++++++++
+
+Follow the GitHub directions to setup a new account.
+
+https://help.github.com/categories/53/articles
+
+If you plan to use ssh to push/pull, setup SSH keys.
+
+https://help.github.com/categories/56/articles
+
+
+
+.. _Configure your local git:
+
+Configure your local git
+++++++++++++++++++++++++
+
+Here is the uncommented version of the commands:
+
+.. code-block:: bash
+
+    git config --global user.name 'Thomas Van Doren'
+    git config --global user.email 'thomas@example.com'
+
+    # linux/mac
+    git config --global core.autocrlf input
+
+    # windows
+    git config --global core.autocrlf true
+
+    # If using ssh keys, verify access to github. It should respond with a
+    # message including your github username.
+    ssh git@github.com
+
+    # Clone your fork of the chapel repo!
+    git clone git@github.com:<github_username>/chapel.git
+
+    # Or, if using HTTPS instead of SSH.
+    git clone https://github.com/<github_username>/chapel.git
+
+    # Set up remotes for github
+    cd chapel
+    git remote add upstream https://github.com/chapel-lang/chapel.git
+    # Make sure it works, get up-to-date without modifying your files
+    git fetch upstream
+    # Change remote for upstream push to "no_push" 
+    git remote set-url --push upstream no_push 
+    # Optionally add remotes for commonly viewed branches
+    git remote add <branch_owner_username> https://github.com/<branch_owner_username>/chapel.git
+
+.. _New branch command:
+
+New branch command
+++++++++++++++++++
+
+.. code-block:: bash
+
+    git checkout -b <branch_name>
+
+.. _Development commands:
+
+Development commands
+++++++++++++++++++++
+
+Stage a file/dir for commit:
+
+.. code-block:: bash
+
+    git add path/to/file
+
+    # (sort of) similar to:
+    svn add path/to/file
```

Comment:
I think it'd be an improvement to put the svn commands in the comment, like
`# (sort of) similar to: svn add path/to/file`
There are a lot more of these if you agree.
My thought here is that one might expect to be able to paste whatever shows up in the code box, and the svn command doesn't help in that context.